spec/features/fixity_dashboard_spec.rb

```diff
@@ -32,6 +32,22 @@
     end
   end
 
+  context "the fixity check for a resource by a cloud service provider is for a deleted resource" do
+    it "displays that the event was a failure" do
```

Comment:
How does this test check that it's in the failure table?

It seems to check for the text "SUCCESS" which is confusing given the title of the test.

Reply:
@hackartisan Thanks for the catch! Fixed.